dealership_auth/index.ts

```diff
@@ -65,18 +65,26 @@ async function initServer() {
       }),
       cookie: {
         maxAge: 1000 * 60 * 60 * 24,
```

Comment:
Does this set the cookie max age to 1000 days? Am I doing my math correctly?

Reply:
this will equal 24 hours. 1000 ms * 60 seconds * 60 minutes * 24 hours

Reply:
Ah, ms. I did my calculation alll wrong. ty.